<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettingsWidget</class>
 <widget class="QWidget" name="ConnectionSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="server_layout">
     <item>
      <widget class="QLabel" name="s_header">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сервер:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="s_ip_layout">
       <item>
        <widget class="QLabel" name="s_ip_label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s_ip_edit">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="s_port_layout">
       <item>
        <widget class="QLabel" name="s_port_label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Порт:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s_port_edit">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="client_layout">
     <item>
      <widget class="QLabel" name="client_header">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Клиент:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="c_ip_layout">
       <item>
        <widget class="QLabel" name="c_id_label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="c_ip_edit">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="c_port_layout">
       <item>
        <widget class="QLabel" name="c_port_label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Порт:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="c_port_edit">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="accept_Btn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_Btn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
